src/Common/src/System/Net/Security/CertificateHelper.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The code is also used by ClientWebSocket (UAP).  There are other certificate related helpers in this folder already.